### ISSUE TITLE
Custom Legend object not working

### ISSUE THREAD

**Issue by gapipro** [hanai/vue-chart.js#2]
Hi,
so if I pass into `legend` prop a settings object its set to `null` when rendering chart.

Problem is that in prop coerce function you are not returning a value by default:
https://github.com/hanai/vue-chart.js/blob/master/src/utils/componentOptionsConstructor.js#L51

Should be:

``` javascript
coerce: function coerce(val) {
 if (typeof val === 'boolean') {
    return { display: val };
 }
 return val;
},
```

**Comment by hanai** [comment]
fixed 62a4522519075fadcfbab9005aa64a51f5ee73fc
please update to 0.0.9

**Comment by gapipro** [comment]
Awesome. TY.
